doc/source/user_guide/boundary_conditions.rst

```diff
@@ -53,25 +59,26 @@ The following example demonstrates how you can define boundary conditions using
         'quit'
     )
 
-Copying Boundary Conditions
+Copying boundary conditions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-define/boundary-conditions/copy-bc TUI: Copies boundary conditions to other zones.
+The ``define.boundary_conditions.copy_bc`` TUI copies boundary conditions to other zones.
```

Comment:
These are meant to be slash separated  with dashes on purpose as it's referring to the Fluent TUI command.  This is how it should look I think:
```suggestion
The ``define/boundary-conditions/copy-bc`` TUI command copies boundary conditions to other zones.
```